src/main/java/io/cryostat/Cryostat.java

```diff
@@ -179,6 +180,8 @@ interface Client {
 
         RecordingMetadataManager recordingMetadataManager();
 
+        MatchExpressionManager matchExpressionManager();
```

Comment:
It doesn't look like the injected instance is getting used here, so I think it can be removed.